index.html

```diff
@@ -18,7 +18,16 @@
         <h1>Welcome to the student showcase!</h1>
     </div>
     <div class="content">
-        <div>ONE</div>
+        <div>
+            <div id="pic">
+                <img src="https://i.imgur.com/ycdXJOt.jpg" alt="Me" height="200px" width="auto">
```

Comment:
Every time you use an inline style a web developer's child is kidnapped and held for ransom. Do you really want to do that to those poor children? _Think of the children_